.github/workflows/daily-boot-iso.yml

```diff
@@ -0,0 +1,42 @@
+name: Build daily Rawhide+COPR boot.iso
+on:
+  schedule:
+    - cron: 0 22 * * *
+  # be able to start this action manually from a actions tab when needed
+  workflow_dispatch:
+
+jobs:
+  boot_iso:
+    name: Build boot.iso
+    runs-on: ubuntu-latest
+    steps:
+      # lorax and its dependencies do not exist in Ubuntu, and we want to test the latest Fedora version anyway
+      # lorax does mounts and uses loop devices, thus needs to be privileged
+      - name: Start fedora container
+        run: |
+          mkdir data
+          docker run --name fedora -d --privileged --network host -v $PWD/data:/data fedora:latest sleep infinity
+
+      - name: Install lorax
+        run: docker exec fedora dnf install -y lorax
+
+      - name: Run lorax
+        run: docker exec -w /data fedora lorax -p Fedora -v 34 -r 34 -s http://download.fedoraproject.org/pub/fedora/linux/development/rawhide/Everything/x86_64/os/ -s https://download.copr.fedorainfracloud.org/results/rpmsoftwaremanagement/dnf-nightly/fedora-rawhide-x86_64/ -s https://download.copr.fedorainfracloud.org/results/@rhinstaller/Anaconda/fedora-rawhide-x86_64/ -s https://copr-be.cloud.fedoraproject.org/results/@storage/blivet-daily/fedora-rawhide-x86_64/ /data/results
+
+      - name: Make generated files accessible
+        run: docker exec fedora chmod -R a+rX /data/
+
+      - name: Upload log artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: logs
+          path: |
+            data/*.log
+            data/*.txt
+
+      - name: Upload image artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: images
+          path: |
+            data/results/**
```

Comment:
I hope there is no quota for file storage. However, let's see how this will work and we can think of other solution later if there will be a problem.

Reply:
I thought about that, too. I set the [workflow retention period](https://github.com/rhinstaller/kickstart-tests/settings/actions) to 7 days for now. If that is somehow not respected, or we need to keep other logs longer, then we could add another workflow which regularly cleans up all boot.iso except for the most two recent ones.

Reply:
Ok, thanks for info.